apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -218,11 +218,25 @@ const BookingPage = ({
     ? booking?.attendees.slice(1).map((attendee) => attendee.email)
     : [];
 
+  // There should only exists one default userData variable for primaryAttendee.
```

Comment:
There were several declaration tries to get user attendee default name, wether it came from booking or query param. With these we only declare/obtain it once.